tests/interface/test_qasm3_utils.py

```diff
@@ -26,8 +26,11 @@
 from qbraid.interface.qbraid_qasm3.tools import (
     convert_to_qasm3,
     qasm3_depth,
-    qasm3_num_qubits,
-    qasm3_qubits,
+    qasm_3_num_qubits,
+    qasm_depth,
+    qasm_num_qubits,
+    qasm_qubits,
```

Comment:
Updating these naming conventions should fix the import errors in the tests
- `qasm_3_num_qubits` $\rightarrow$ `qasm3_num_qubits`
- `qasm_depth` $\rightarrow$ `qasm3_depth`
- `qasm_num_qubits` $\rightarrow$ `qasm3_num_qubits`
- `qasm_qubits` $\rightarrow$ `qasm3_qubits`

Reply:
Thanks!